stdlib/LinearAlgebra/src/uniformscaling.jl

```diff
@@ -240,11 +240,11 @@ end
 function isapprox(J::UniformScaling, A::AbstractMatrix;
                   atol::Real = 0,
                   rtol::Real = Base.rtoldefault(promote_leaf_eltypes(A), eltype(J), atol),
-                  nans::Bool = false, norm::Function = vecnorm)
+                  nans::Bool = false, norm::Function = norm)
     n = checksquare(A)
-    normJ = norm === LinearAlgebra.norm ? abs(J.λ) :
-            norm === vecnorm   ? abs(J.λ) * sqrt(n) :
-                                 norm(Diagonal(fill(J.λ, n)))
+    normJ = norm === opnorm ? abs(J.λ) :
+            norm === norm   ? abs(J.λ) * sqrt(n) :
```

Comment:
is this an issue?

Reply:
You are right, I didn't recognise the potential disagreement with `opnorm(J::UniformScaling, p::Real=2) = opnorm(J.λ, p)`. So, there are two possibilities:

1. Change `opnorm(J::UniformScaling, p::Real=2) = abs(J.λ)` as suggested by @stevengj and leave this line as it is.
2. Change this part to `norm === opnorm ? opnorm(J.λ) : `

What do you suggest to do?

Reply:
I would suggest just leaving `abs`.   Changing this to use `opnorm(J.λ)` is unrelated to this PR and can be addressed elsewhere if needed.

Reply:
I actually was talking about the 'norm===norm' part. Won’t that always be 'true'?

Reply:
Should be `norm === Base.norm`

Reply:
Thank you very much. I've fixed it.